File: Python/Manual 04/Question05.py
n = int(input("Enter the number of rows: "))

for i in range(n - 2, n + 1):
    for k in range(1, n - i + 1):
        print(" ", end="")

    for j in range(1, n + 1):
        if j <= i:
            print("* ", end="")
        else:
            print("  ", end="")

    for k in range(i + 1, n + 1):
        print("  ", end="")

    for j in range(1, i + 1):
        print("* ", end="")

    print()

for i in range(1, 2 * n + 1):
    for k in range(1, i + 1):
        print(" ", end="")

    for j in range(1, 2 * n - i + 1):
        print("*", end="")

    for j in range(1, 2 * n - i + 1):
        print("*", end="")

    print()

File: JavaScript/Manual 01/Question05.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter an alphabet: ", function (alphabet) {
    if (!"abcdefghijklmnopqrstuvwyz".includes(alphabet.toLowerCase())) {
        console.log("Invalid Input");
        rl.close();
        return;
    }
    if (alphabet == 'a' || alphabet == 'e' || alphabet == 'i' || alphabet == 'o' || alphabet == 'u')
        console.log("The alphabet is a Vowel.");
    else
        console.log("The alphabet is a Consonant.");
    rl.close();
});